mixnode/src/node/node_description.rs

```diff
@@ -0,0 +1,40 @@
+use serde::Deserialize;
+use serde::Serialize;
+use std::fs::File;
+use std::path::PathBuf;
+use std::{fs, io};
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub struct NodeDescription {
+    pub(crate) name: String,
+    pub(crate) description: String,
+    pub(crate) link: String,
+}
+
+impl Default for NodeDescription {
+    fn default() -> Self {
+        NodeDescription {
+            name: "This node has not yet set a name".to_string(),
+            description: "This node has not yet set a description".to_string(),
+            link: "https://nymtech.net".to_string(),
+        }
+    }
+}
+
+impl NodeDescription {
+    pub(crate) fn load_from_file(mut config_path: PathBuf) -> io::Result<NodeDescription> {
```

Comment:
should this (and `save_to_file`) be given `config_path` as arguments? Should it rather be `node_description_path`? Also, as I've mentioned before, I think it should be part of config itself.